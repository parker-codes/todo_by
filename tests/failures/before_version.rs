use todo_by::todo_while_version;

fn main() {
    todo_while_version!("<0.1.0");
    fn my_function() {}
}
