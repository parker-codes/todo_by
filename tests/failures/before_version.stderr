error: TODO b4 >0.1.0 not satisfied
 --> tests/failures/before_version.rs:4:5
  |
4 |     todo_b4!(">0.1.0");
  |     ^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `todo_b4` (in Nightly builds, run with -Z macro-backtrace for more info)
